streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsProducerTest.java

```diff
@@ -254,30 +224,30 @@ public void shouldNotCloseProducerIfEosDisabled() {
 
     @Test
     public void shouldInitTxOnEos() {
-        assertTrue(eosMockProducer.transactionInitialized());
+        assertThat(eosMockProducer.transactionInitialized(), is(true));
     }
 
     @Test
     public void shouldBeginTxOnEosSend() {
         eosStreamsProducer.send(record, null);
-        assertTrue(eosMockProducer.transactionInFlight());
+        assertThat(eosMockProducer.transactionInFlight(), is(true));
     }
 
     @Test
     public void shouldContinueTxnSecondEosSend() {
         eosStreamsProducer.send(record, null);
         eosStreamsProducer.send(record, null);
-        assertTrue(eosMockProducer.transactionInFlight());
-        assertThat(eosMockProducer.uncommittedRecords().size(), equalTo(2));
+        assertThat(eosMockProducer.transactionInFlight(), is(true));
+        assertThat(eosMockProducer.uncommittedRecords().size(), is(2));
     }
 
     @Test
     public void shouldForwardRecordButNotCommitOnEosSend() {
         eosStreamsProducer.send(record, null);
-        assertTrue(eosMockProducer.transactionInFlight());
-        assertTrue(eosMockProducer.history().isEmpty());
-        assertThat(eosMockProducer.uncommittedRecords().size(), equalTo(1));
-        assertThat(eosMockProducer.uncommittedRecords().get(0), equalTo(record));
+        assertThat(eosMockProducer.transactionInFlight(), is(true));
```

Comment:
I was wondering what's the standard for using `assertThat` vs `assertTrue`? Do we have a convention to follow?

Reply:
`assertThat` is nicer in general, but it doesn't really matter. In this case, IDEA offered to translate, and I was already changing a lot of assertions, so I just accepted the translation.